CHANGELOG.md

```diff
@@ -1,47 +1,12 @@
 # Change Log
 
-## [3.11.0](https://github.com/auth0/wp-auth0/tree/3.11.0) (2019-05-30)
-[Full Changelog](https://github.com/auth0/wp-auth0/compare/3.10.0...3.11.0)
+## 4.0.0-beta
+[Full Changelog](https://github.com/auth0/wp-auth0/compare/3.11.0...master)
```

Comment:
sounds weird an ongoing changelog pointing to master? Maybe save this for the release, or take note to fix this before releasing?

Also note you are replacing the `3.11.0` notes instead of appending one on top of it.

Reply:
I wanted to keep a public log of changes in case anyone asks what's different in `master`. Having it in the changelog seemed like the best place. 

That said ... I have **no** idea why I replaced 3.11.0 ... and didn't notice! Thanks for that